### PR TITLE
Feature/ Create form component

### PR DESCRIPTION
**Pull request by JennyDoJD**
- Design
image

- Result
image

-Ticket: [Create components](https://trello.com/c/bvrOqz8S/42-create-components)

### CODE REVIEW

File: practice/src/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/main.css">
+  <title>Products Management</title>
+</head>
+
+<body>
+  <!-- Edit Feature -->
```
Comment (ghost) [inline practice/src/index.html]: @JennyDoJD I saw duplicate HTML code for the product form. Could you help double-check and find the best way to deal with it?

File: practice/src/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <div class="form-button">
+          <button class="btn btn-primary btn-confirm" type="reset">Cancel</button>
+          <button class="btn btn-secondary btn-confirm" type="submit">Save</button>
+        </div>
+      </div>
+    </form>
+  </div>
+
+  <!-- Delete Feature -->
+  <form action="javascript:void(0)" class="form-wrapper form-delete">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: This is modal, not form

File: practice/src/index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+<body>
+  <!-- Edit Feature -->
+  <div class="form-wrapper">
+    <h2 class="secondary-heading form-heading-confirm">Edit</h2>
+    <form action="javascript:void(0)" class="form-modal">
+      <div class="form-info">
+        <div class="form-group">
+          <label for="editName" class="main-title">Name</label>
+          <input type="text" class="input-form" required />
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Don't use `required` attribute in HTML to validate. Please remove it. We will use JS to validate form

File: practice/src/index.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          <label for="editPrice" class="main-title">Price</label>
+          <input type="number" class="input-form" required />
+          <label for="editImage" class="main-title">Image URL</label>
+          <input type="url" class="input-form" required />
+          <label for="editQuantity" class="main-title">Quantity</label>
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should wrap label and input into group. You can refer https://getbootstrap.com/docs/4.0/components/forms/

File: practice/src/css/components/form.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+.form-button {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 8px;
+}
+
+.btn-confirm {
+  width: 175px;
```
Comment (huyencao-agilityio) [inline practice/src/css/components/form.css]: Should use padding here

File: practice/src/css/components/form.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
+.form-wrapper {
+  border-radius: 16px 16px 0;
+  width: 445px;
```
Comment (huyencao-agilityio) [inline practice/src/css/components/form.css]: Can we use padding instead of width?

File: practice/src/css/components/form.css [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  padding: 40px 50px 30px 50px;
+  background-color: var(--primary-background-color);
+}
+
+.form-heading-feature {
+  padding-bottom: 24px;
+  border-bottom: 1px solid var(--tertiary-border-color);
+  width: 70%;
```
Comment (huyencao-agilityio) [inline practice/src/css/components/form.css]: No need set width here

Comment (JennyDoJD) [reply practice/src/css/components/form.css]: image
I want the bottom border to be like in Figma so I set the width here

**Pull request merged**